SSA/Projects/LLVMRiscV/Evaluation/LLVM-tests/add-imm64-to-sub.lean

```diff
@@ -0,0 +1,268 @@
+import SSA.Projects.LLVMRiscV.Pipeline.InstructionLowering
+
+open LLVMRiscV
+/-
+define i64 @add_b31(i64 %x) {
+; NOZBS-LABEL: add_b31:
+; NOZBS:       # %bb.0:
+; NOZBS-NEXT:    lui a1, 524288
+; NOZBS-NEXT:    sub a0, a0, a1
+; NOZBS-NEXT:    ret
+;
+; ZBS-LABEL: add_b31:
+; ZBS:       # %bb.0:
+; ZBS-NEXT:    bseti a1, zero, 31
+; ZBS-NEXT:    add a0, a0, a1
+; ZBS-NEXT:    ret
+  %add = add i64 %x, 2147483648
+  ret i64 %add
+}
+-/
+-- add_b31
+def add_b31_llvm_i64 := [LV| {
+    ^entry (%x: i64):
+    %0 = llvm.mlir.constant (2147483648) : i64
```

Comment:
In LLVM, these %0 would be indented. (holds for everything below)

Reply:
indentation fixed.